### PR TITLE
feat: Quantity update feature

### PR DESCRIPTION
**Pull request by manojncsoni007**
screenshot:
image

### CODE REVIEW

File: src/pages/Cart/Cart.css [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
 .product-info {
     padding: 1rem;
-    width: 300px;
+    width: 350px;
```
Comment (HeyNitin) [inline src/pages/Cart/Cart.css:45]: use rems here too. maintain CSS consistency

File: src/pages/Cart/Cart.jsx [hunk 2]
```diff
@@ -1,11 +1,20 @@
 import React from 'react';
 import './Cart.css';
 import { Navbar } from '../../components'
-import { useCart } from '../../context';
-import { showToast } from '../../utils/toast';
+import { useAuth, useCart } from '../../context';
+import { addToWishlist, removeFromCart, updateCartQuantity } from '../../services';
+import { getTotalCartPrice } from '../../utils';
```
Comment (HeyNitin) [inline src/pages/Cart/Cart.jsx:6]: You can use absolute imports from next project

**Review by HeyNitin — APPROVED** [review]
Left some comments, UI looks clean, you can proceed to merge

File: src/pages/Cart/Cart.jsx (hunk 2 above)
Comment (manojncsoni007) [reply src/pages/Cart/Cart.jsx:6]: will do

**Pull request merged**